### DISCUSSION TITLE
[bsc#1233161] Put focus on the search box

### DISCUSSION THREAD

**Discussion by shundhammer** [agama-project/agama#1813] [Ideas]
_Moved here from [bsc#1233161](https://bugzilla.suse.com/show_bug.cgi?id=1233161) by [Heitor da Silva](mailto:heitormoreira@yahoo.com.br):_

> Whenever possible, put the focus on the search box so the user can search immediately.

![searchbox](https://github.com/user-attachments/assets/7d046f7a-944b-4515-8100-5ee0adbc5aaf)

**Comment by shundhammer** [comment]
@dgdavid answered:

> Thanks for the suggestion, Heitor!
>
> While I'm not sure that focusing on the search box right away is the best approach, it's true that we do have some focus-related issues to address, as mentioned in https://github.com/agama-project/agama issues/1521.
>
> As for the search box, although it's likely to be a frequently used widget, I need to do a bit more research to determine whether immediately focusing there is a good idea from an accessibility perspective. Another option could be to support the widely used Ctrl+K keyboard shortcut, but we'll need to evaluate that option as well since it may be more appropriate for global search.
>
> We definitely need to take some time to think this through before implementing any solution. Please be patient; I'll let you know once we have made improvements.
>
> Regards.

**Comment by dgdavid** [comment]
I still having doubts about placing focus automatically in such an input from an a11y point of view.

In any case, I foresee changes in how L10n settings will be managed in a near future, since most probably them will be moved/included in a more generic "System page" called to replace the "Overview" one.

Thus, I'm going to close this discussion by now. Feel free to re-open it if you consider I should re-evaluate its priority.
